setup.py

```diff
@@ -28,9 +28,9 @@
     install_requires=install_requires,
     entry_points={
         "prefect.collections": [
-            "AwsCredentials = prefect_aws.credentials",
-            "MinIOCredentials = prefect_aws.credentials",
-            "S3Bucket = prefect_aws.s3",
+            "prefect_aws = prefect_aws",
+            "prefect_aws_s3 = prefect_aws.s3",
+            "prefect_aws_ecs = prefect_aws.ecs",
```

Comment:
Curious, what does changing these lines do?

Reply:
Discussed this with Alex, but basically the plugin registration wasn't used quite as intended. We should be registering a collection or a module in a collection rather than specific objects. See these debug logs from the core library

Before 
```
❯ PREFECT_DEBUG_MODE=1 ipython -c "import prefect"
Loaded collection 'AwsCredentials'.
Loaded collection 'MinIOCredentials'.
Loaded collection 'S3Bucket'.
```

After

```
❯ PREFECT_DEBUG_MODE=1 ipython -c "import prefect"
Loaded collection 'prefect_aws'.
Loaded collection 'prefect_aws_ecs'.
Loaded collection 'prefect_aws_s3'.
```

Reply:
Additional discussion in https://github.com/PrefectHQ/prefect-aws/pull/85#discussion_r961742877

Reply:
Yeah, moving forward we should have all blocks available at the root level for smaller collections so that we can have only one entrypoint, and one entrypoint per module in larger collections like `prefect-aws` or `prefect-gcp`. This will remove the need for most contributors to add an entrypoint.